course/lesson-22-functions-return-values/lesson.tres

```diff
@@ -154,7 +154,7 @@ In previous lessons, we had characters walking on grids.
 
 And for those practices, you were working directly with cell coordinates.
 
-Well, cell coordinates don't correspond to positions on the screen, so we need to convert cell positions to screen positions."
+Well, cell coordinates don't correspond to positions on the screen. Each cell have a size in pixels of 120x120, so to get the center of any cell, we need to convert it's cell position to a screen position."
```

Comment:
```suggestion
Well, cell coordinates don't correspond to positions on the screen. To find the center of any cell on the screen, we need to convert the cell's coordinates to a position on the screen, in pixels."
```

Reply:
Fixing two grammar errors:

- `each` is singular -> `each cell has`
- `it's position` -> `its position` because it's a possessive pronoun and not the verb to be we're using here.

I recommend running your edits through a grammar checker. You could use one in emacs, though options are pretty limited. A simple solution is to copy paste your writings into [hemingway](https://hemingwayapp.com/) or google docs.

Reply:
That's a good idea, I'll start double checking my writing with Google Docs.